tests/integration/helpers/app-version-test.js

```diff
@@ -11,15 +10,13 @@ test('it displays only app version', function(assert) {
 
   this.render(hbs`{{app-version hideSha=true}}`);
 
-  assert.ok(this.$().text().match(versionRegExp), 'Displays version.');
-  assert.ok(!this.$().text().match(shaRegExp), 'Does not display git sha.');
+  assert.equal(this.$().text(), '1.2.3', 'Displays version.');
 });
 
 test('it displays only git sha', function(assert) {
   assert.expect(2);
 
   this.render(hbs`{{app-version hideVersion=true}}`);
 
-  assert.ok(this.$().text().match(shaRegExp), 'Displays git sha.');
-  assert.ok(!this.$().text().match(versionRegExp), 'Does not display version.');
+  assert.equal(this.$().text(), 'deadbeef', 'Displays git sha.');
```

Comment:
Maybe I'm missing something, but why is this asserting that the text matches `deadbeef`?